ledger-tool/src/blockstore.rs

```diff
@@ -327,11 +327,11 @@ pub fn blockstore_subcommands<'a, 'b>(hidden: bool) -> Vec<App<'a, 'b>> {
             .arg(&starting_slot_arg)
             .arg(&ending_slot_arg)
             .arg(
-                Arg::with_name("target_ledger")
-                    .long("target-ledger")
+                Arg::with_name("output_dir")
+                    .long("output-dir")
```

Comment:
I'm open to changing `target`. However, I do feel more strongly about keeping `ledger` in the flag name. If there is already a Blockstore within `--target-ledger`, then the shreds will be inserted into it. The use of `ledger` within the flag name reinforces this / reinforces that this isn't just some regular directory